src/Microsoft.OData.Edm/Schema/EdmAnnotationsTarget.cs

```diff
@@ -0,0 +1,56 @@
+//---------------------------------------------------------------------
+// <copyright file="EdmAnnotationsTarget.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.OData.Edm
+{
+    /// <summary>
+    /// Represents an EDM annotations target.
+    /// </summary>
+    public class EdmAnnotationsTarget : IEdmAnnotationsTarget

```

Comment:
>EdmAnnotationsTarget [](http://example.com/codeflow?start=17&length=20)

should we override the equals operator to say that two instances are equal if all of their segments are equal?